### PR TITLE
Fix messed up tests for dropout

### LINKED ISSUE #22109: Multiple (redundant?) return statements in test_jit.py
In https://github.com/pytorch/pytorch/blob/master/test/test_jit.py#L5900-L5945 there are several methods with multiple return statements like below:

```python
@torch.jit.script_method
def forward(self, x):
    x = self.conv(x)
    return x
    x = self.bn(x)
    return F.relu(x, inplace=True)
```

It seems they are redundant, as subsequent return statements are unreachable. If they are there for a debugging purpose, a comment would be nice.

### PR DESCRIPTION
**Pull request by Chillee**
Fix https://github.com/pytorch/pytorch/issues/22109.

I've confirmed with @suo that this wasn't intentional.

### CODE REVIEW

**Review by suo — APPROVED** [review]
thanks!

**Comment by Chillee** [comment]
I don't think these look like my fault...
